docs/index.md

```diff
@@ -1 +1,80 @@
---8<-- "README.md"
+# Frequenz Channels for Python
+
+## Introduction
+
+Frequenz Channels is a *channels* implementation for Python.
+
+According to [Wikipedia](https://en.wikipedia.org/wiki/Channel_(programming)):
+
+> A channel is a model for interprocess communication and synchronization via
+> message passing. A message may be sent over a channel, and another process or
+> thread is able to receive messages sent over a channel it has a reference to,
+> as a stream. Different implementations of channels may be buffered or not,
+> and either synchronous or asynchronous.
+
+Frequenz Channels are mostly designed after [Go
+channels](https://tour.golang.org/concurrency/2) but it also borrows ideas from
+[Rust channels](https://doc.rust-lang.org/book/ch16-02-message-passing.html).
+
+## Installation
+
+First, you need to make sure you have Python installed (at least version 3.11):
+
+```console
+$ python3 --version
+Python 3.11.4
+```
+
+!!! note
+
+    These instructions assume you are using a [POSIX compatible
+    `sh`](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/sh.html)
+    shell.
+
+If that command doesn't print a version newer than 3.11.0, you'll need to
+[download and install Python](https://www.python.org/downloads/) first.
```

Comment:
the instructions rely on having `pip` and `venv` already installed, and these modules might not be installed if the user  only install python3.11. So it might be worth mentioning the installation of `pip` and `venv` is also needed

Reply:
You are right about `pip`, but `venv` is a [built-in Python module](https://docs.python.org/3/library/venv.html), are you sure it is not installed by default? That would be very weird...

Reply:
Added link to instruction on how to install `pip`.

Reply:
extracted from https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/
 ```
Installing virtualenv
Note If you are using Python 3.3 or newer, the [venv](https://docs.python.org/3/library/venv.html#module-venv) module is the preferred way to create and manage virtual environments. venv is included in the Python standard library and requires no additional installation.
```

Reply:
I remembered stumbling on that issue in the past too, but I think it is a problem with how multiple python versions are installed in Ubuntu., when installing from sources for sure it comes with venv. Also, if you are in Ubuntu, it looks like it tells you how to fix it. On the other hand, using venv is not technically necessary, it is just a recommendation.

Reply:
> Also, if you are in Ubuntu, it looks like it tells you how to fix it. On the other hand, using venv is not technically necessary, it is just a recommendation.

right, I have a similar thought about it.